### ISSUE TITLE
Add trip reset button

### ISSUE THREAD

**Issue by RobertJansen1** [CoMPaTech/stromer#77]
### Describe the bug.

Hi, with much joy have i used the integration so far and i was wondering if it is possible to add a button to reset the trip distance and such through the integration

### What version of Home Assistant Core has the issue?

NA

### What type of installation are you running?

Home Assistant OS

### How did you install the stromer integration

HACS

**Comment by CoMPaTech** [comment]
That should be possible, was already starting to reply/drafting code on it but life got in the way, let me see what I can cook up. As per maintenance I can't check on my own bike currently, but I will make a 'beta' release that you should be able to fetch via HACS.

**Comment by CoMPaTech** [comment]
@RobertJansen1 https://github.com/CoMPaTech/stromer/releases/tag/v0.3.2a1 seems to provide the button and correctly pick up the name (it said 'restart' in `a0`. If you enable 'beta' releases (using 'download' from the three dots) on the Stromer card in HACS you should be able to test if it yields the requested result. (As indicated I unfortunately can't test at this moment)

**Comment by RobertJansen1** [comment]
Thanks for the fast patch! i tested it out and unfortunattely, for me this is not working. the butten registers an action in Home Assitant, but it seems not to be send through to the bike. i enabled debug logging, but the code does not show any debug logs from the integration (i expect the python debug lines to be in there somewhere, but nothing)

**Comment by CoMPaTech** [comment]
I'll recheck the bearings then ... probably something with the path it's sending to. I'll update accordingly and see if we can stuff more debug in.

**Comment by CoMPaTech** [comment]
Not sure why the (actual code) debug logs don't pass, don't spot them here either, will look into that. But I made a small mistake in the API path (`/id/` missing from `/bike/id/{bikeid}/trip_data`). Could you give https://github.com/CoMPaTech/stromer/releases/tag/v0.3.2a2 a go?

**Comment by CoMPaTech** [comment]
@RobertJansen1 did you get a chance to test it with the newer version?

**Comment by RobertJansen1** [comment]
Yes i did test it and no it is still not working. I wanted to test if the python library you mentioned in the other issue is able to reset the trip, but I wasn't able to do that yet. I will try that today hopefully and report back

**Comment by CoMPaTech** [comment]
Hmmm, strange, but indeed I referenced from that python library indeed ... hopefully bike is back this week so I can test myself as well.

**Comment by RobertJansen1** [comment]
With the original library, it does reset the trip. So the call there seems the right one :) I will double check the home assistant one again later this week (it is now reset)

**Comment by jlsjonas** [comment]
Hi, did either of you manage to test it since? Wouldn't mind assisting if still required!

**Comment by CoMPaTech** [comment]
Confirmed myself it didn't work last week but didn't (yet) see the difference between the calls (referring to `elnkoscs`' module). Busy otherwise so didn't make time to further check

**Comment by CoMPaTech** [comment]
Stupid mistake in the code - but I've now reset it in debugging - after ride home we'll know if https://github.com/CoMPaTech/stromer/releases/tag/v0.3.2a5 works

**Comment by CoMPaTech** [comment]
Just confirming that it works, will create a new issue to properly handle errors. I.e. when you try to reset the trip but the bike is offline, HA will throw an error.

**Comment by RobertJansen1** [comment]
Thanks very much for the effort, I confirmed as well that this now works.
